### DISCUSSION TITLE
Cannot download files

### DISCUSSION THREAD

**Discussion by LesDeuxAmis** [aclap-dev/video-downloadhelper#781] [Help and Q&A]
### Your question

Hello,
I cannot download this file
https://mixdrop.club/f/rweq17eohv8do3q/2/_SUB_ITA_The_Zone_of_Interest_2023_WEBRip_AC3_CB01.mp4
The download seems to be over in just a few seconds, but the size of the dowloaded file is only 1KB and it obviously doesn't work.

Can you help me?

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: it
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "download_directory": "C:\\Users\\admin\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\admin\\Downloads"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 1,
    "successfull_dl": 2
}

**Comment by LesDeuxAmis** [comment]
I realized only now that the link wasn't visible... now it is ;)

**Reply by GrampaWildWilly** [reply]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/076983fa-f43e-4434-acf0-14015803fceb)

Maybe the content timed out.  It's just not there now.

**Reply by LesDeuxAmis** [reply]
https://mixdrop.club/f/wn1gnjlktg377w/2/_SUB_ITA_The_Zone_of_Interest_2023_WEBRip_AC3_CB01.mp4

Here's the new link. Same problem.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

Strange case.  I went to the page & got an error trying to download the video in VDH.  I looked in the Network Monitor but there was not a lot of useful information there.  There were no manifests & no media objects.  But there was one HTML object with a name that ended in .mp4.  That seems particularly pathological.  I double clicked that entry & Firefox opened the same page again in its own tab.  I popped up the context menu on the video & saw that there was a cascade submenu for a frame.  On the submenu, I executed Open Frame in New Tab.  That opened the video by itself in its own tab in the native built-in video player of Firefox.  Popping up the context menu on that was not helpful.  So I launched playback & stopped it after a couple of seconds.  The popup context menu now contained a Save Video As entry.  But executing that resulted in a length-0 mp4 file.

I think this site has very intentionally done what is necessary to thwart downloading.  They have not implemented DRM itself.  But they've done essentially the same thing.

Since this content is not protected by DRM, I tried to record a short section of it with OBS.  That worked fine.  So failing downloading the content, you can record it.  I recently looked for screen recorders that would record content protected by DRM & could not find any.  It seems that screen recorders are as blocked by DRM protection as are downloaders.  But this content is not protected by DRM.  So even though they've quite effectively blocked downloading, you can record it.  I am not endorsing OBS in any way.  I'm sure there's lots of other screen recorders that would record this content just as well.  There's a screen recorder built into Windows 11 that is part of the . . . I can't remember what they call it.  Clipchimp or something like that.  I'm sure that would work, too, although I haven't bothered to try it.

**Reply by Mattmjs** [reply]
I can tell you why it doesn't work and it has nothing to do with VDH. The speed is so terribly throttled under 100KB/s even below
50KB/s that the download cancels itself. Just move on to another site, Mixdrop is not worth your time.
